Apps/W1/HybridGP/app/src/Migration/Support/HelperFunctions.codeunit.al

```diff
@@ -941,181 +941,17 @@ Codeunit 4037 "Helper Functions"
-        GLItemLedgerRelation: Record "G/L - Item Ledger Relation";
-        GLRegister: Record "G/L Register";
-        Location: Record Location;
-        TrackingSpecification: Record "Tracking Specification";
-        ReservationEntry: Record "Reservation Entry";
-        ItemJournalLine: Record "Item Journal Line";
-        PostValueEntryToGL: Record "Post Value Entry to G/L";
-        BankAccount: Record "Bank Account";
-        BankAccountPostingGroup: Record "Bank Account Posting Group";
-        BankAccountLedgerEntry: Record "Bank Account Ledger Entry";
-        BankAccReconciliation: Record "Bank Acc. Reconciliation";
-        BankAccReconciliationLine: Record "Bank Acc. Reconciliation Line";
-        PurchaseHeader: Record "Purchase Header";
-        PurchaseLine: Record "Purchase Line";
-        OverReceiptCode: Record "Over-Receipt Code";
-        AccountingPeriod: Record "Accounting Period";
-    begin
-        GPConfiguration.DeleteAll();
-        GLEntry.DeleteAll(true);
-        GLRegister.DeleteAll(true);
-        CustLedgerEntry.DeleteAll(true);
-        DetailedCustLedgEntry.DeleteAll(true);
-        Customer.DeleteAll(true);
-        PurchaseLine.ModifyAll("Qty. Rcd. Not Invoiced", 0);
-        PurchaseLine.DeleteAll(true);
-        PurchaseHeader.DeleteAll(true);
-        VendorLedgerEntry.DeleteAll(true);
-        DetailedVendorLedgEntry.DeleteAll(true);
-        Vendor.DeleteAll(true);
-        ItemLedgerEntry.DeleteAll(true);
-        AvgCostAdjmtEntryPoint.DeleteAll(true);
-        ValueEntry.DeleteAll(true);
-        PostValueEntryToGL.DeleteAll(true);
-        TrackingSpecification.DeleteAll(true);
-        ReservationEntry.DeleteAll(true);
-        ItemJournalLine.DeleteAll(true);
-        Item.DeleteAll(true);
-        ItemUnitOfMeasure.DeleteAll(true);
-        GLItemLedgerRelation.DeleteAll(true);
-        ResetGLDimensionSetup();
-        DimensionSetEntry.DeleteAll(true);
-        DimensionValue.DeleteAll(true);
-        Dimension.DeleteAll(true);
-        PaymentTerms.DeleteAll(true);
-        PaymentTermTranslation.DeleteAll(true);
-        DataMigrationEntity.DeleteAll();
-        Location.DeleteAll(true);
-        ItemTrackingCode.DeleteAll(true);
-        BankAccountLedgerEntry.DeleteAll(true);
-        BankAccount.DeleteAll(true);
-
-        if OverReceiptCode.Get('GP') then
-            OverReceiptCode.Delete(true);
-
-        BankAccountPostingGroup.Reset();
-        BankAccountPostingGroup.SetFilter(Code, PostingGroupCodeTxt + '*');
-        if not BankAccountPostingGroup.IsEmpty() then
-            BankAccountPostingGroup.DeleteAll();
-
-        BankAccReconciliationLine.DeleteAll(true);
-        BankAccReconciliation.DeleteAll(true);
-
-        DataMigrationStatus.Reset();
-        DataMigrationStatus.SetRange("Migration Type", GetMigrationTypeTxt());
-        if not DataMigrationStatus.IsEmpty() then
-            DataMigrationStatus.DeleteAll();
-
-        CleanupGenJournalBatches();
```

Comment:
Should CleanupGenJournalBatches and CleanupVatPostingSetup procedure code be removed too?

Reply:
I don't know what those are for, but they seem outside the scope of this work item. Do you know why they are there?

Reply:
They were meant to remove any 'GP' batches from prior migration runs. Since we removed the call to the procedure I don't believe we need the procedure any longer. At least I didn't find any other calls to the procedure.

Reply:
Updated, thanks